sdk/monitor/azure-monitor-ingestion/CHANGELOG.md

```diff
@@ -1,20 +1,20 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.2.0 (2024-04-12)
```

Comment:
Should there be any extra entries added to the TROUBLESHOOT.md related to these new capabilities?

Reply:
Considering the nature of what was added, I think adding a sample showing how to set `LogIngestionAudience` and then a troubleshoot entry pointing to the sample makes sense. I'll add those to this PR.